### ISSUE TITLE
Error creating new Talent project

### ISSUE THREAD

**Issue by PhilippMundhenk** [GENIVI/iot-event-analytics#92]
When creating a new talent project according to prerequisites and tldr, I end up with 
```
ENOENT: No such file or directory, scandir '<iotea_path>/src/sdk/javascript/lib'
```
Which I guess is correct, the lib folder does not exist...

**Comment by llakie** [comment]
See `yarn sdk.build` or `yarn sdk.get` for further information. Will be fixed in #88

**Comment by PhilippMundhenk** [comment]
If fixed in #88, can close here, I guess.
